.github/workflows/package_integration_test.yaml

```diff
@@ -9,6 +9,7 @@ on:
       - 'cmd/cli/plugin/package/**'
       - 'cmd/cli/plugin/secret/**'
       - 'tkg/tkgctl'
+      - 'all_tests.txt'
```

Comment:
was debating whether we should make the file hidden, but decided it is a reasonable price to pay to keep as is if it means making this mechanism more discoverable.